metrics.py

```diff
@@ -2,26 +2,17 @@
 (https://github.com/jamesmullenbach/caml-mimic/blob/master/evaluation.py)
 They are used for the internal need to compare with CAML."""
 
-import numpy as np
-
-def intersection(y_true, y_pred, axis):
-    # y_true and y_pred are #data * #labels matrices
-    return np.logical_and(y_pred, y_true).sum(axis=axis).astype(float)
 
-def macro_precision(y_true, y_pred):
-    precision_per_label = intersection(y_true, y_pred, 0) / (y_pred.sum(axis=0) + 1e-10)
-    return np.mean(precision_per_label)
+import numpy as np
+from sklearn.metrics import precision_recall_fscore_support
 
-def macro_recall(y_true, y_pred):
-    recall_per_label = intersection(y_true, y_pred, 0) / (y_true.sum(axis=0) + 1e-10)
-    return np.mean(recall_per_label)
 
 def macro_f1(y_true, y_pred):
-    prec = macro_precision(y_true, y_pred)
-    rec = macro_recall(y_true, y_pred)
+    prec, rec, _, _ = precision_recall_fscore_support(y_true, y_pred, average='macro')
```

Comment:
I now think we can do macro_prec, macro_rec to clearly see this is the average of prec or rec across labels